docs/claudie-workflow/claudie-workflow.md

```diff
@@ -29,189 +28,193 @@
-  //Get all configs from database
+  // Gets all configs from the database.
   rpc GetAllConfigs(GetAllConfigsRequest) returns (GetAllConfigsResponse);
-  // Sets the manifest to null which forces the deletion of the infra,
-  // defined by the manifest, next time when the config will be picked up.
+  // Sets the manifest to null, effectively forcing the deletion of the infrastructure
+  // defined by the manifest on the very next config (diff-) check.
   rpc DeleteConfig(DeleteConfigRequest) returns (DeleteConfigResponse);
-  // Deletes config from database.
+  // Deletes the config from the database.
   rpc DeleteConfigFromDB(DeleteConfigRequest) returns (DeleteConfigResponse);
 ```
 ### Flow
-
-- Receives config file from Frontend calculates `msChecksum` and saves it to the database
-- Periodically push config where `msChecksum` != `dsChecksum` to the schedulerQueue
-- Periodically push config where `dsChecksum` != `csChecksum` to the builderQueue
-- Receives config with desiredState from Scheduler
+- Receives a `config` from Frontend, calculates its `msChecksum` and saves it to the database
+- Periodically checks for `config` changes and pushes the `config` to the `schedulerQueue` if `msChecksum` != `dsChecksum`
+- Periodically checks for `config` changes and pushes the `config` to the `builderQueue` if `dsChecksum` != `csChecksum`
+- Receives a `config` with the `desiredState` from Scheduler
 - Checks if `dsChecksum` == `msChecksum`
-    - `true` -> saves it to the database
-    - `false` -> it ignores this config
-- Receives config with currentState from Builder and saves it to the database
+  - `true` ⇒ ignores this `config`
+  - `false` ⇒ saves the `config` to the database
+- Receives a `config` with `currentState` from Builder and saves it to the database
 - Checks if `csChecksum` == `dsChecksum`
-    - `true` -> saves it to the database
-    - `false` -> it ignores this config
+  - `true` ⇒ ignores this `config`
+  - `false` ⇒ saves the `config` to the database
 
-##### `msChecksum` - manifest checksum
-##### `dsChecksum` - desired state checksum
-##### `csChecksum` - current state checksum
+### Variables used
+| variable     | meaning                |
+| ------------ | ---------------------- |
+| `msChecksum` | manifest checksum      |
+| `dsChecksum` | desired state checksum |
+| `csChecksum` | current state checksum |
 
 
 ## Scheduler
-Scheduler creates desired state of the infra based on the manifest taken from the config received from Context-box.
+Scheduler brings the infrastructure to a desired the state based on the manifest contained in the config that is received from Context-box.
 
-Scheduler also monitors health of the current infra and manages any operations based on it, e.g. replacement of broken node, etc. *[work in progress]*
+Scheduler also monitors the health of current infrastructure and manages any operations based on actual health state (e.g. replacement of broken nodes, etc. *[work in progress]*).
```

Comment:
cc @Despire @MiroslavRepka 
there's a "work in progress" note here - do you guys know if we want to add sth here or can the note be deleted or do we want to keep it?

Reply:
it's still WIP as part of the autoscaler & self-healing features that are yet to come

Reply:
thanks @MarioUhrik.